src/database/session_context.rs

```diff
@@ -43,6 +50,21 @@ impl SessionContextTrait for SessionContext {
             }
         }
     }
+
+    async fn delete_by_token(&self, token_type: TokenType, token: String) -> Result<Model, DbErr> {
```

Comment:
Delete `use crate::entities::session::Model;`
```suggestion
    async fn delete_by_token(&self, token_type: TokenType, token: String) -> Result<session::Model, DbErr> {
```